src/main/java/com/pingcap/tikv/codec/LongUtils.java

```diff
@@ -65,7 +67,7 @@ public static void writeULongFull(CodecDataOutput cdo, long lVal, boolean compar
      * @param lVal The data to encode
      */
     public static void writeLong(CodecDataOutput cdo, long lVal) {
-        cdo.writeLong(CodecUtil.flipSignBit(lVal));
+        cdo.writeLong(TableCodec.flipSignBit(lVal));
```

Comment:
The purpose of this PR is just rename `CodecUtil`.  It would be better not mixing refactoring and renaming together. What do you think?